css-scroll-snap-2/Overview.bs

```diff
@@ -288,21 +229,34 @@ The 'scroll-start-target' property {#scroll-start-target}
 			Animation type: none
 		</pre>
 
-		Values are for the {{scroll-start-target}} shorthand have the same definitions
-		as values for the [[#scroll-start-target-longhands|longhands]].
+		<dl dfn-type=value dfn-for="scroll-start-target">
+			<dt><dfn>none</dfn>
+			<dd>Element is not a ''scroll-start-target''.
+			<dt><dfn>auto</dfn>
+			<dd>Element is a ''scroll-start-target'' and is used to determine the
+				<a>initial scroll position</a> of its nearest ancestor <a>scroll container</a>
+		</dl>
 
 	<h4 id="scroll-start-target-with-place-content">Interaction with <a>place-content</a></h4>
 
-	If the <a>initial scroll position</a> indicated by <a spec="css-align" lt="place-content">place-content</a> differs
+	If the <a>initial scroll position</a> indicated by [[css-align-3#place-content|place-content]] differs
 	from that indicated by {{scroll-start-target}}, user-agents should give preference
-	to the latter.
-
-	<h4 id="cases-where-scroll-start-target-is-inactive">Cases where scroll-start-target is inactive</h4>
-
-	<h5 id="scroll-start-fragment-navigation">Fragment Navigation</h5>
-
-	If an element within the <a>scrollable overflow area</a> of a <a>scroll container</a> is the
-	document’s “:target”, {{scroll-start-target}} is inactive for that <a>scroll container</a>.
+	to the position indicated by {{scroll-start-target}}.
+
+	<h4 id="scroll-start-fragment-navigation">Interaction with Fragment Navigation</h4>
+
+	<a href="https://html.spec.whatwg.org/#try-to-scroll-to-the-fragment">
+	Scrolling to the fragment for a Document</a> might set the <a>scroll position</a> of a
+	<a>scroll container</a>. This might <a spec="cssom-view-1" lt="scroll an element">scroll</a>
+	the scroll container away from its <a>initial scroll position</a>.
+	It might occur that an element which is a {{scroll-start-target}}	for that <a>scroll container</a>
+	is loaded into the document after such a
+	<a href="https://html.spec.whatwg.org/#try-to-scroll-to-the-fragment">
+	scroll to the fragment</a> has occurred. In such an instance, the
+	{{scroll-start-target}} does set the <a>initial scroll position</a> of the
+	<a>scroll container</a>	but does not affect the current <a>scroll position</a> of the
+	<a>scroll container</a> i.e. does not cause it to be scrolled in response to
+	the late arrival of the {{scroll-start-target}}.
```

Comment:
Sorry I think this is a little vague and should be rewritten to say when this will scroll to the scroll-start-target. I.e. if a scroll-start-target is discovered and we have not scrolled to a fragment. We don't need to  mention the initial scroll position part, since scrolling to a fragment doesn't change the initial position but just scrolls away from the initial position, but we do want scroll-start-target to scroll to late discovered targets as long as we haven't done a fragment scroll.

Reply:
Okay, I've tried to make it a bit clearer by pointing to the html spec on [updating the document](https://html.spec.whatwg.org/#updating-the-document) to cover arrivals of scroll-start-targets after the first layout of a scroll container.